src/git/remotes/remoteProviders.ts

```diff
@@ -76,6 +76,12 @@ const builtInProviders: RemoteProviders = [
 	},
 ];
 
+const cloudRemotesMap: { [key: string]: typeof GitHubRemote | typeof GitLabRemote | typeof BitbucketServerRemote } = {
```

Comment:
Nit for stricter typing
```suggestion
const cloudRemotesMap: Record<CloudSelfHostedIntegrationId, typeof GitHubRemote | typeof GitLabRemote | typeof BitbucketServerRemote> = {
```

Reply:
@d13 

Fixed: 9d18d8bdb9efd6f652816489783f73c901e83530